softserve/templates/form.html

```diff
@@ -41,14 +41,6 @@ <h3>Create Nodes</h3>
        </small></p>
     </div>
   </div>
-  <div class='row'>
```

Comment:
Shouldn't this be autopopulated with the key you fetched?

Reply:
Then a user can pass the key there itself if not present on github. Is this the good reason to auto-populate key field?

Reply:
What if they want to use a different key for just one time? :)